cryptography-core/src/asset_proofs/macros.rs

```diff
@@ -74,3 +74,21 @@ macro_rules! define_sub_assign_variants {
         }
     };
 }
+
+/// That `ensure` does not transform into a string representation like `failure::ensure` is doing.
```

Comment:
Which `ensure`?

Reply:
very good question :-? I moved this here from a lib.rs don't have any idea what it is referring to.